slobrok/src/vespa/slobrok/server/local_rpc_monitor_map.h

```diff
@@ -126,6 +129,9 @@ class LocalRpcMonitorMap : public MapListener,
     void addLocal(const ServiceMapping &mapping,
                   std::unique_ptr<ScriptCommand> inflight);
 
+    /** for use by unregister API, returns true on success */
+    bool removeLocal(const ServiceMapping &mapping);
```

Comment:
consider using wouldConflict in the consensus map instead to get this bool (is this a 'clean' remove)